--- qbittorrent-2.5.0rc4/src/lang/qbittorrent_hu.ts~	2010-11-28 15:12:20.000000000 +0100
+++ qbittorrent-2.5.0rc4/src/lang/qbittorrent_hu.ts	2010-12-02 09:47:28.799842667 +0100
@@ -2324,7 +2324,7 @@
     <message>
         <location filename="../preferences/options.ui" line="626"/>
         <source>Append the label of the torrent to the save path</source>
-        <translation>A címke hozzfűzése a letöltési könyvtár nevéhez</translation>
+        <translation>A címke hozzáfűzése a letöltési könyvtár nevéhez</translation>
     </message>
     <message>
         <location filename="../preferences/options.ui" line="636"/>
