--- qbittorrent-2.5.0beta4/src/downloadthread.cpp~	2010-11-16 21:34:31.000000000 +0100
+++ qbittorrent-2.5.0beta4/src/downloadthread.cpp	2010-11-18 10:59:47.776944667 +0100
@@ -100,7 +100,7 @@
 }
 
 void downloadThread::loadCookies(const QString &host_name, QString url) {
-  const QList<QByteArray> raw_cookies = RssSettings::getHostNameCookies(host_name);
+  const QList<QByteArray> raw_cookies = RssSettings().getHostNameCookies(host_name);
   QNetworkCookieJar *cookie_jar = networkManager.cookieJar();
   QList<QNetworkCookie> cookies;
   qDebug("Loading cookies for host name: %s", qPrintable(host_name));
--- qbittorrent-2.5.0beta4/src/mainwindow.cpp~	2010-11-17 18:55:08.000000000 +0100
+++ qbittorrent-2.5.0beta4/src/mainwindow.cpp	2010-11-18 11:09:11.217493335 +0100
@@ -207,7 +207,7 @@
   // View settings
   actionTop_tool_bar->setChecked(pref.isToolbarDisplayed());
   actionSpeed_in_title_bar->setChecked(pref.speedInTitleBar());
-  actionRSS_Reader->setChecked(RssSettings::isRSSEnabled());
+  actionRSS_Reader->setChecked(RssSettings().isRSSEnabled());
   actionSearch_engine->setChecked(pref.isSearchEnabled());
   displaySearchTab(actionSearch_engine->isChecked());
   displayRSSTab(actionRSS_Reader->isChecked());
@@ -1197,7 +1197,7 @@
 }
 
 void MainWindow::on_actionRSS_Reader_triggered() {
-  RssSettings::setRSSEnabled(actionRSS_Reader->isChecked());
+  RssSettings().setRSSEnabled(actionRSS_Reader->isChecked());
   displayRSSTab(actionRSS_Reader->isChecked());
 }
 
