--- qbittorrent-3.3.11/m4/qbittorrent.m4.orig	2017-05-29 09:55:47.560076042 +0200
+++ qbittorrent-3.3.11/m4/qbittorrent.m4	2017-05-29 09:57:34.790498191 +0200
@@ -12,10 +12,10 @@
                                 [QT_QMAKE=`AS_DIRNAME(["$QT_QMAKE"])`])
                  ])
 
-AS_IF([test -f "$QT_QMAKE/qmake"],
-              [QT_QMAKE="$QT_QMAKE/qmake"],
-              [AS_IF([test -f "$QT_QMAKE/qmake-qt4"],
-                             [QT_QMAKE="$QT_QMAKE/qmake-qt4"],
+AS_IF([test -f "$QT_QMAKE/qmake-qt4"],
+              [QT_QMAKE="$QT_QMAKE/qmake-qt4"],
+              [AS_IF([test -f "$QT_QMAKE/qmake"],
+                             [QT_QMAKE="$QT_QMAKE/qmake"],
                              [QT_QMAKE=""])
               ])
 
@@ -36,10 +36,10 @@
                                 [host_bins])
                  ])
 
-AS_IF([test -f "$QT_QMAKE/qmake"],
-              [QT_QMAKE="$QT_QMAKE/qmake"],
-              [AS_IF([test -f "$QT_QMAKE/qmake-qt5"],
-                             [QT_QMAKE="$QT_QMAKE/qmake-qt5"],
+AS_IF([test -f "$QT_QMAKE/qmake-qt5"],
+              [QT_QMAKE="$QT_QMAKE/qmake-qt5"],
+              [AS_IF([test -f "$QT_QMAKE/qmake"],
+                             [QT_QMAKE="$QT_QMAKE/qmake"],
                              [QT_QMAKE=""])
               ])
 
